.github/workflows/release-automation.yml

```diff
@@ -0,0 +1,47 @@
+name: Update flyteidl version
```

Comment:
Lets add  a section to the Contributing guide ,on how to test github workflow changes

Reply:
should we add this? Currently, we only have act but I am not sure act will work for everyone. i do my testing in my private repo but I will not recommend that

Reply:
@evalsocket  would be good if we can add current approach for the community to contribute to changes in these actions. 
Lets add what you do for testing it right now and iterate and add the recommended approach as we find the best practices.